tables/subject_history/ikemura21-pth.csv

```diff
@@ -0,0 +1,7 @@
+,ref_name,subject_name,event,event_date_cmp,event_date,location,iso_name,vaccine_name,severity
+0,Ikemura21,B.1.1.7_infection,isolation,~,2021-01-31,Japan,NULL,NULL,NULL
+1,Ikemura21,B.1.1.7_infection,isolation,~,2021-01-31,Japan,NULL,NULL,NULL
+2,Ikemura21,B.1.617.2_infection,isolation,~,2021-10-31,Japan,NULL,NULL,NULL
+3,Ikemura21,B.1.617.2_infection,isolation,~,2021-10-31,Japan,NULL,NULL,NULL
+4,Ikemura21,BNT162b2,isolation,~,2021-10-31,Japan,NULL,NULL,NULL
```

Comment:
BNT162b2 should have event in the sequence as `1st dose`, `2nd dose`, `2nd dose isolation`